### PR TITLE
Update interview questions

### PR DESCRIPTION
**Pull request by oandrz**
Add some new interview questions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1070,6 +1078,7 @@ It is also a good practice to annotate overridden methods with `@Override` to ma
 
 * Firebase. [Firebase.google.com](https://firebase.google.com/)
 
+* Leak Canary [Droidcon NYC 2015 About Leak Canary by Py](https://www.youtube.com/watch?v=mU1VcKx8Wzw)
```
Comment (MaTriXy) [inline README.md]: I would add the actual link 
https://square.github.io/leakcanary/

Comment (oandrz) [reply README.md]: okay sure

**Comment by oandrz** [comment]
>>I would add a proper link to leakCanary and it's usage - not just a video

The reason I put that video is because that video explained by the one who made the leak canary itself and it give you explanation about the basic of how you use leak canary

File: README.md (hunk 1 above)
Comment (oandrz) [reply README.md]: add it

**Pull request closed without merging**